src/main/java/io/castle/client/internal/backend/OkRestApiBackend.java

```diff
@@ -226,12 +225,12 @@ public void onResponse(Call call, Response response) throws IOException {
     }
 
     @Override
-    public void sendPrivacyRemoveUser(String userId) {
+    public Response sendPrivacyRemoveUser(String userId) {
```

Comment:
I think it would make more sense to return a boolean instead of the response object.